Domains/4-Application/ISM/Released/IntegratedStructuralModel.02.00.00.ecschema.xml

```diff
@@ -0,0 +1,2755 @@
+        <ECEnumerator value="2200" name="TimberUnspecified" displayLabel="Timber Unspecified" description="Timber Unspecified." />
+    </ECEnumeration>
+
+    <ECEnumeration typeName="IsmCurveMemberUse" backingTypeName="int" isStrict="true" description="Curve Member Use.">
+        <ECEnumerator value="100" name="Beam" displayLabel="Beam" description="Beam." />
+        <ECEnumerator value="200" name="Column" displayLabel="Column" description="Column." />
+        <ECEnumerator value="300" name="HorizontalBrace" displayLabel="Horizontal Brace" description="Horizontal Brace." />
+        <ECEnumerator value="400" name="StripFooting" displayLabel="Strip Footing" description="Strip Footing." />
+        <ECEnumerator value="500" name="Pile" displayLabel="Pile" description="Pile." />
+        <ECEnumerator value="600" name="VerticalBrace" displayLabel="Vertical Brace" description="Vertical Brace." />
+        <ECEnumerator value="450" name="Pedestal" displayLabel="Pedestal" description="Pedestal." />
+    </ECEnumeration>
+
+    <ECEnumeration typeName="IsmMemberLoadResistance" backingTypeName="int" isStrict="true" description="Load Resistance.">
+        <ECEnumerator value="100" name="GravityOnly" displayLabel="Gravity Only" description="Gravity Only." />
+        <ECEnumerator value="200" name="GravityAndLateral" displayLabel="Gravity and Lateral" description="Gravity and Lateral." />
+        <ECEnumerator value="300" name="LateralOnly" displayLabel="Lateral Only" description="Lateral Only." />
+        <ECEnumerator value="400" name="None" displayLabel="None" description="None." />
+    </ECEnumeration>
+
+    <ECEnumeration typeName="IsmCartesianGridLineKind" backingTypeName="int" isStrict="true" description="Ism Grid Line Kind.">
+        <ECEnumerator value="20" name="R" displayLabel="R" description="R." />
+        <ECEnumerator value="40" name="S" displayLabel="S" description="S." />
+    </ECEnumeration>
+
+    <PropertyCategory typeName="IsmMember_category" displayLabel="Member" priority="190" />
+    <ECEntityClass typeName="IsmMember" displayLabel="Ism Member" modifier="Abstract" >
+        <BaseClass>bis:PhysicalElement</BaseClass>
+        <BaseClass>IIsmObject</BaseClass>
+        <ECProperty propertyName="LoadResistance" typeName="IsmMemberLoadResistance" displayLabel="Load Resistance" category="IsmMember_category" description="Member load resistance." />
+    </ECEntityClass>
+
+    <ECEntityClass typeName="IsmSpanningMember" displayLabel="Ism Spanning Member" modifier="Abstract">
+        <BaseClass>IsmMember</BaseClass>
+    </ECEntityClass>
+
+    <ECEnumeration typeName="ReactionLimitState" backingTypeName="int" isStrict="true" description="Ism Grid Line Kind.">
+        <ECEnumerator value="10" name="Service" displayLabel="Service" description="Service." />
+        <ECEnumerator value="20" name="Ultimate" displayLabel="Ultimate" description="Ultimate." />
```

Comment:
unnecessary display labels and descriptions

Reply:
removed